src/components/Blockquote/Blockquote.docs.mdx

```diff
@@ -0,0 +1,32 @@
+import { Story, Preview } from '@storybook/addon-docs/blocks';
+import Blockquote from './Blockquote';
+
+# Blockquote
+
+A stylized blockquote that accentuates a quote from a larger body of text. Renders an HTML `<blockquote>` tag.
+
+## Usage
+
+```js
+import { Blockquote } from '@quartz/interface/src/components';
+```
+
+## Props
+
+<Props of={Blockquote} />
+
+## Usage Guidelines
+- **Do** wrap a `Blockquote` component to change the text style or constrain its width.
```

Comment:
I don't think we want to encourage people to change the text style, unless I'm misunderstanding this

Reply:
Oh I just brought this over from qz-react and I think it may be aimed at avoiding context props. I can remove.